npm/vue/src/index.ts

```diff
@@ -417,7 +415,7 @@ export function mountCallback (
   options: any = {},
 ): () => Cypress.Chainable {
   return () => {
-    return mount(component, options)
+    throw new Error('mountCallback has been removed')
```

Comment:
Same here, let's as least recommend how to use `mount` and point somewhere useful. Maybe "`mountCallback` has been deprecated. Consider using `mount` with a `beforeEach` hook to mount your components" or something, perhaps?

Reply:
https://github.com/cypress-io/cypress/pull/24419/commits/e8af254b28368c07e270dab9c6ea0b7a92e12387